rca/programmes/models.py

```diff
@@ -707,6 +707,20 @@ def get_context(self, request, *args, **kwargs):
         except EmptyPage:
             subpages = paginator.page(paginator.num_pages)
 
+        # Listing filters
+        degree_levels = [i.title for i in DegreeLevel.objects.all()]
+        degree_levels_title = DegreeLevel._meta.verbose_name.capitalize()
```

Comment:
Oh that’s pretty cool, didn’t think we could do that!

Reply:
Yea - it's unlikely to change, but if it does, at least we only change it in one place